src/components/Dashboard/ThisWeek/WeeklyActivity/WeeklyActivity.tsx

```diff
@@ -143,113 +158,100 @@ const WeeklyActivity = ({ accountListId }: Props): ReactElement => {
-                <StyledTableCell
-                  align="right"
-                  data-testid="WeeklyActivityTableCellMessagesThatProducedAppointments"
-                >
-                  {loading || !data ? (
-                    <Skeleton variant="text" />
-                  ) : (
-                    numberFormat(
-                      data.messagesThatProducedAppointments.totalCount,
-                      locale,
-                    )
-                  )}
-                </StyledTableCell>
-              </TableRow>
-              <TableRow>
-                <StyledTableCell>{t('Appointments')}</StyledTableCell>
-                <StyledTableCell
-                  align="right"
-                  data-testid="WeeklyActivityTableCellCompletedAppointments"
-                >
-                  {loading || !data ? (
-                    <Skeleton variant="text" />
-                  ) : (
-                    numberFormat(data.completedAppointments.totalCount, locale)
-                  )}
-                </StyledTableCell>
-                <StyledTableCell></StyledTableCell>
-              </TableRow>
-              <TableRow>
-                <StyledTableCell>{t('Correspondence')}</StyledTableCell>
-                <StyledTableCell
-                  align="right"
-                  data-testid="WeeklyActivityTableCellCompletedCorrespondence"
-                >
-                  {loading || !data ? (
-                    <Skeleton variant="text" />
-                  ) : (
-                    numberFormat(
-                      data.completedCorrespondence.totalCount,
-                      locale,
-                    )
-                  )}
-                </StyledTableCell>
-                <StyledTableCell></StyledTableCell>
-              </TableRow>
-            </TableBody>
-          </Table>
-        </TableContainer>
+        <CardContent
+          className={[classes.cardContent, classes.cardContentExpanded].join(
+            ' ',
+          )}
+        >
+          <TableContainer component="div" className={classes.tableContainer}>
+            <Table size="small">
+              <TableHead>
+                <TableRow>
+                  <StyledTableCell data-testid="WeeklyActivityTableCellDateRange">
+                    {`${intlDateFormat(interval.start)} - ${intlDateFormat(
+                      interval.end,
+                    )}`}
+                  </StyledTableCell>
+                  <StyledTableCell align="right">
+                    {t('Completed')}
+                  </StyledTableCell>
+                </TableRow>
+              </TableHead>
+              <TableBody>
+                <TableRow>
+                  <StyledTableCell>{t('Initiations')}</StyledTableCell>
+                  <StyledTableCell
+                    align="right"
+                    data-testid="WeeklyActivityTableCellCompletedInitiations"
+                  >
+                    {loading || !data ? (
```

Comment:
If the data is in the cache, `data` will be truthy by `loading` will also be true. All of these rows check `loading`, which means that it won't use the cached data when its available.